grunt/mochacli.js

```diff
@@ -3,7 +3,7 @@ module.exports = function() {
     return {
         options: {
             files: ['test/integration/*.js', 'test/unit/*.js'],
-            require: ['test/utils/mock-coinstac-pouch-config.js'],
+            require: ['test/init.js'],
```

Comment:
because init will  load the polyfill.  doing the polyfill then the pouch config still error'd :/